.gitignore

```diff
@@ -16,6 +16,7 @@
 #===============================================================================
 
 build
+build_acl
```

Comment:
This looks like a personalised build folder. What is the reason for adding this, and not having cmake nest build folders inside a parent build folder which is already ignored?

Reply:
Hi @theComputeKid,
I have restored gitignore, thanks.

Reply:
I think it should be safe to use `build*` in .gitignore if devs want to use multiple build folders prefixed with build.

Reply:
Agree completely with @mgouicem